### PR TITLE
Using Inversify with hapi.js

### PR DESCRIPTION
**Pull request by Talento90**
A simple example how to integrate inversify with hapi.

### CODE REVIEW

File: hapi/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "typescript-node",
```
Comment (remojansen) [inline hapi/package.json]: Can you update this please maybe call it something like "inversify-hapi-example"

File: hapi/package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "license": "MIT",
+  "repository": {
+    "url": "https://github.com/Talento90/typescript-node.git"
```
Comment (remojansen) [inline hapi/package.json]: Can you please update to  https://github.com/inversify/inversify-code-samples.git

**Comment by remojansen** [comment]
Hi @Talento90 thanks a lot for this PR I have added some comments to the code review can you please take a look? thanks!

**Comment by Talento90** [comment]
Sorry, copy&paste... I have done all the changes :+1:

**Comment by remojansen** [comment]
Great, thanks a lot! I will try to see if some things can be improved in the example because I want IoC to be the most important part of the example. Maybe we need to add more controllers and services...

**Pull request merged**